pkg/commands/subscription/list_test.go

```diff
@@ -69,7 +69,7 @@ func TestSubscriptionList(t *testing.T) {
 		assert.NilError(t, err)
 		ol := strings.Split(out, "\n")
 		assert.Check(t, util.ContainsAll(ol[0], "NAME", "CHANNEL", "SUBSCRIBER", "REPLY", "DEAD LETTER SINK", "READY", "REASON"))
-		assert.Check(t, util.ContainsAll(ol[1], "s0", "InMemoryChannel:imc0", "ksvc:ksvc0", "broker:b00", "broker:b01"))
+		assert.Check(t, util.ContainsAll(ol[1], "s0", "InMemoryChannel:imc0", "ksvc0", "broker:b00", "broker:b01"))
```

Comment:
@cardil this `ksvc` prefix was dropped because of changes in `SinkToString` -> `AsText()` functions. It might have been midly useful to have a type reference in this list. But not necessary required.

Reply:
Yes. I wrote the `AsText` method to be as close as possible to the old one, but I thought that if we default to kservice in the input (`ksvc:`, and `kservice:` prefixes are optional), maybe we should also be consistent on the output side?

If you think, we should print those prefixes, even for kservice, then I can change it back.

BTW. Maybe we should deprecate the `SinkAsString` method (in favor of `AsText`), as it doesn't care about namespaces. If you target something from a different namespace, it will just print it by name, which is confusing to say the least.

Reply:
I see, OK let's keep it as optional. Apparently, judging by the test changes we haven't expected a lot of those types prefixes anyway.